### PR TITLE
Patch #227: Output path defaults to xlsxfile<directory>.xml

### LINKED ISSUE #227: output_path should default to path/to/file
## Expected behavior
When trying to convert a file, if output_path is not supplied, output_path should automatically be set to the same directory of the file being converted, using the same file name (other than changing the extension to .xml).

**Example**
`path/to/myFile.xls` --> `path/to/myFile.xml`

## Actual behavior
output_path is a required argument

### PR DESCRIPTION
**Pull request by p-netm**
- makes the output_path cli command optional
- output path defaults to the xlsx file input directory but with a filename extension of '.xml' 
    i.e. 

```python
xls2xform /some/path/to/file.xlsx  # output-path is not defined
``` 
will write the file converted file to -> /some/path/to/file.xml

this fixes #227

### CODE REVIEW

File: requirements.pip [hunk 1]
```diff
@@ -2,14 +2,14 @@
 # This file is autogenerated by pip-compile
 # To update, run:
 #
-#    pip-compile --output-file requirements.pip requirements.in
+#    pip-compile --output-file=requirements.pip requirements.in
 #
 argparse==1.4.0           # via unittest2
 formencode==1.3.1
-functools32==3.2.3.post2; python_version < "3.2"
```
Comment (ukanga) [inline requirements.pip]: While we still run tests for python 2.7 this is a required package. Please do not remove it.

File: pyxform/xls2xform.py [hunk 2]
```diff
@@ -4,13 +4,17 @@
...
+from os.path import splitext
 from pyxform import builder, xls2json
 from pyxform.utils import has_external_choices, sheet_to_csv
 """
 xls2xform converts properly formatted Excel documents into XForms for
 use with ODK Collect.
 """
 
+def get_xml_path(path):
+    """takes in xslxform path and returns the output-path for the same file"""
```
Comment (ukanga) [inline pyxform/xls2xform.py]: I am being picky here but any sentence starting in lower case is not appropriate. https://www.pythonforbeginners.com/basics/python-docstrings/

Comment (p-netm) [reply pyxform/xls2xform.py]: refactored

**Comment by p-netm** [comment]
issue resolved in pr #302 whose commit history is much cleaner

**Pull request closed without merging**